browse/static/js/connectedpapers.js

```diff
@@ -0,0 +1,116 @@
+(function () {
+  var $output = $('#connectedpapers-output');
+
+  if ($output.html() != '') {
+    // Toggled off
+    $output.html('');
+    return;
+  }
+
+  $output.html('<p>Loading...</p>');
+
+  const REST_ADDR = 'https://rest.migration.connectedpapers.com/';
+  const CONNECTED_PAPERS_ADDR = 'https://www.connectedpapers.com/';
+  const ARXIV_THUMBNAILS_ADDR = CONNECTED_PAPERS_ADDR + 'arxiv_thumbnails/';
+  const NUMBER_OF_THUMBNAILS = 18;
+  
+  var arxivId = window.location.pathname.split('/').reverse()[0];
+  var arxivIdToCPIdUrl = REST_ADDR + '?arxiv=' + arxivId;
+  var communicationErrorHtml = '<p>Oops, seems like communication with the Connected Papers server is down.</p>';
+  var idNotRecognizedHtml = '<p>Seems like this paper is still not in our database. Please try again in a few days.</p>';
+  
+  
+  $.get(arxivIdToCPIdUrl).done(translationResponse => {
+    if ($output.html() == '') {
+      // Toggled off
+      return;
+    }
+    if (translationResponse == null) {
+      $output.html(idNotRecognizedHtml);
+      return;
+    }
+    var paperId = translationResponse.paperId;
+    var title = translationResponse.title;
+
+    if (paperId.length == 0 || title.length == 0) {
+      $output.html(idNotRecognizedHtml);
+      return;
+    }
+
+    var versionsFetchUrl = REST_ADDR + '?versions=' + paperId;
+
+    $.get(versionsFetchUrl).done(versionsResponse => {
+      if ($output.html() == '') {
+        // Toggled off
+        return;
+      }
+
+      var graphUrl = CONNECTED_PAPERS_ADDR + 'main/' + paperId + '/arxiv';
+      var buildGraphLinkHtml = '<a href="' + graphUrl + '" target="_blank"><p style="margin:0;">View graph for ' + 
+                                title + '</p></a>';
+      var seeGraphLinkHtml = '<a href="' + graphUrl + '" target="_blank"><p style="margin:0;">View graph for ' +
+                              title + '</p></a>';
+      var graphNotVisual = '<p>Seems like ' + title + ' is still not in our database. Please try again in a few days.</p>';
```

Comment:
Suggest not using the title in this 'not found' message, as arXiv paper titles can be very long! The message might get lost. "Seems like this paper is still not in our database" seems sufficient to me.

Reply:
Also, if there is an option for users to submit papers this would be a good place to add that link.

Reply:
Thanks for the suggestion Shamsi - I agree. Itay, can you please update the microcopy?

Re submitting papers - we don't have this option currently.

Reply:
Done.